daemon/rpc/handlers.go

```diff
@@ -86,8 +88,67 @@ func (server *Server) SwapIn(ctx context.Context, req *SwapInRequest) (*SwapInRe
 }
 
 func (server *Server) SwapOut(ctx context.Context, req *SwapOutRequest) (*SwapOutResponse, error) {
-	log.Info("HELLO WORLD")
-	log.Infof("Received SwapOut request: %v", req)
+	log.Info("Swapping out")
+
+	// Validate request
+	if req.AmountSats <= 0 {
+		return nil, fmt.Errorf("amount must be greater than 0")
+	}
```

Comment:
this is `uint64` at this point, so we can remove it

Reply:
true